specviz/widgets/workspace.py

```diff
@@ -351,13 +351,15 @@ def compose_filter_string(reader):
 
         # Include an auto load function that lets the io machinery find the
         # most appropriate loader to use
-        loader_name_map['Auto (*)'] = None
-
-        # This ensures that users actively have to select a file type before
-        # being able to select a file. This should make it harder to
-        # accidentally load a file using the wrong type, which results in weird
-        # errors.
-        filters = ['Select loader...'] + list(loader_name_map.keys())
+        auto_filter = 'Auto (*)'
+        loader_name_map[auto_filter] = None
+
+        filters = list(loader_name_map.keys())
+        # Make sure that the "Auto (*)" loader shows up first. Being a bit
+        # pedantic about this even though we can probably just rely on
+        # dictionary ordering here.
+        index = filters.index(auto_filter)
+        filters = filters[index:index+1] + filters[:index] + filters[index+1:]
```

Comment:
Just a bit of bikeshedding, but I find `filters = filters.insert(0, filters.pop(index))` a _smidge_ clearer. 🤓

Reply:
Doh 🤦‍♂️